### DISCUSSION TITLE
Sending Bitcoin with the BTC pay server demo vers

### DISCUSSION THREAD

**Discussion by solutionprovidertoproblems** [btcpayserver/btcpayserver#5767] [Technical Support]
I'm new to BTC Pay server. I need some help as I'm stuck. I sent my bitcoin to the demo version on https://mainnet.demo.btcpayserver.org.  Subsequently, I initiated steps to transfer some of it to make some online payments but I couldn't proceed further. Will it still be possible to send the funds or use it before I am able to do a full or complete set up with deployment?

**Comment by pavlenex** [comment]
@ndeet has already replied on Mattermost chat, so leaving here for visibility and closing it:

> When you setup a store there and the bitcoin wallet you only had the option of entering an xpub or watch-only wallet, which means a wallet that you control, so the funds should still be with you and not with the demo server.
